src/test/kotlin/no/nav/faktureringskomponenten/controller/FakturaserieControllerIT.kt

```diff
@@ -399,3 +525,9 @@ class FakturaserieControllerIT(
             )
         ).serialize()
 }
+
+interface FakturaserieRepositoryForTesting : JpaRepository<Fakturaserie, String> {
```

Comment:
En kommentar om at dette er for å unngå bruk av `@Transactional`?

Reply:
ja det kan jeg legge inn, kanskje ta opp bruk av transactional i komptester på teknisk også

Reply:
Jeg tror det er bra å ta dette opp :+1: